packages/config-utils/proxy.js

```diff
@@ -39,9 +97,12 @@ module.exports = ({
   bounceProd = false,
   useAgent = true,
   useDevBuild = true,
+  localApps = process.env.LOCAL_APPS,
+  defaultLocalAppHost = process.env.LOCAL_APP_HOST || 'stage.foo.redhat.com',
```

Comment:
The `defaultLocalAppHost` can also be `prod.foo.redhat.com`. It should be adjusted based on the env.

Reply:
@Hyperkid123 The `defaultLocalAppHost` is now dependent on the `majorEnv` and should work for prod and stage now.